.github/workflows/ci.yml

```diff
@@ -72,3 +72,8 @@ jobs:
           command: bench
           args: --target ${{ matrix.test-target || matrix.target }} --verbose --no-run ${{ matrix.args }}
         if: matrix.rust == 'nightly'
+
+  minimal-versions:
+    uses: RustCrypto/actions/.github/workflows/minimal-versions.yml@master
+    with:
+        working-directory: ${{ github.workflow }}
```

Comment:
Argh, you can not use the shared config. You can use the following config instead:
```yaml
  minimal-versions:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: RustCrypto/actions/cargo-cache@master
      - uses: actions-rs/toolchain@v1
        with:
          toolchain: nightly
          override: true
          profile: minimal
      - uses: RustCrypto/actions/cargo-hack-install@master
      - name: Test benches build
        run: cargo build --benches
      - run: cargo update -Z minimal-versions
      - run: cargo hack test --release --feature-powerset
```